tsconfig.json

```diff
@@ -6,12 +6,13 @@
     "outDir": "./dist/out-tsc",
     "sourceMap": true,
     "declaration": false,
-    "module": "esnext",
+    "module": "es2020",
```

Comment:
Is the net effect of this no change?

Reply:
As it stands I don't believe so.  It now reverted